Deploy vm without postfix number in name

Koleon opened the pull request:
This will allow to create VM's hostname and name without postfix number when var.vm_count == 1.

Commit:
Fix typo in filenames of sphere_virtual_machine_bare.tf, vsphere_virtual_machine_linux.tf, vsphere_virtual_machine_windows.tf

Koleon committed:
Merge pull request #1 from Koleon/fix-typo-filenames

Fix typo in filenames of vsphere_virtual_machine_*.tf

Commit:
Deploy VM without postfix number in VM's name and hostname

Antoine98 commented:
Hello,
Can you modify to : 
name             = "${var.vm_name_prefix}${var.vm_count == 1 ? "" : "-${count.index}"}"

Without the hyphen when var.vm_count == 1.
Thank you

Koleon committed:
Without the hyphen when var.vm_count == 1

As Antoine98 mention [here](https://github.com/vancluever/terraform-vsphere-virtual-machine/pull/3#issuecomment-461627555)

Koleon committed:
Without the hyphen when var.vm_count == 1

As Antoine98 mention [here](https://github.com/vancluever/terraform-vsphere-virtual-machine/pull/3#issuecomment-461627555)

Koleon committed:
Without the hyphen when var.vm_count == 1

As Antoine98 mention [here](https://github.com/vancluever/terraform-vsphere-virtual-machine/pull/3#issuecomment-461627555)

Koleon commented:
@Antoine98 You are totally right. Changes were made. Thank you!

vancluever commented:
I haven't maintained this module in a very long time, and I don't have the vSphere infra to test any changes on, so I'm closing this in preparation of archival.

If you found the module useful, feel free to fork it!
